client/components/backup-storage-space/usage-display.tsx

```diff
@@ -0,0 +1,56 @@
+import { ProgressBar } from '@automattic/components';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { getRewindBytesAvailable, getRewindBytesUsed } from 'calypso/state/rewind/selectors';
+import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
+import { useStorageUsageText } from './hooks';
+import { StorageUsageLevels } from './storage-usage-levels';
+
+const PROGRESS_BAR_CLASS_NAMES = {
+	[ StorageUsageLevels.Full ]: 'red-warning',
+	[ StorageUsageLevels.Critical ]: 'red-warning',
+	[ StorageUsageLevels.Warning ]: 'yellow-warning',
+	[ StorageUsageLevels.Normal ]: 'no-warning',
+};
+
+type OwnProps = {
+	loading?: boolean;
+	usageLevel: StorageUsageLevels;
+};
+
+const UsageDisplay: React.FC< OwnProps > = ( { loading = false, usageLevel } ) => {
+	const siteId = useSelector( getSelectedSiteId ) as number;
+
+	const translate = useTranslate();
+
+	const bytesAvailable = useSelector( ( state ) => getRewindBytesAvailable( state, siteId ) );
+	const bytesUsed = useSelector( ( state ) => getRewindBytesUsed( state, siteId ) );
+	const storageUsageText = useStorageUsageText( bytesUsed, bytesAvailable );
+	const loadingText = translate( 'Calculating…', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Calculating price' )` *ES Score: 8*